tests/test_monkey_patch/test_sphinx_output_basic_elba_off_napoleon_rtype_inline_autodoc_description_.txt

```diff
@@ -0,0 +1,17 @@
+Example Module
+**************
+
+example.function_with_return_type_annotations(a, b)
```

Comment:
Can you give some context about what's going on with all of these files?

Reply:
Yep sure (and again, I wasn't sure whether this was the best solution so happy to change). The rationale is this:

I found it really hard to tell the difference just by looking at the results within the tests (using breakpoints and stuff). I  stumbled across https://pytest-regressions.readthedocs.io/en/latest/index.html, which seemed a not terrible way to keep track of what is produced when each of the different configurations are run. 

For each of the tests, one file like this is produced. That allows the developer/user/anyone looking at the repo to see what is actually produced when each different set of config is used. Each time the tests are run, the output is compared to the file in the repo. If they match, the test passes, if they don't match, the test fails. If we make a change and want to update them all at once, that can be done with [a simple flag](https://pytest-regressions.readthedocs.io/en/latest/overview.html#regen-all). The name is automatically taken from the pytest parametrization which is not super easy to read, but it is possible to map each file to the test that produced it (although there are probably better ways). 

The regression aspect of the tests and all these files could be removed and the tests would still pass/fail (at the moment I've blended the regression and unit/integration testing which is a bit lazy). The reason I added it was so I could see all the different permutations of what can be produced based on different config (if we got really fancy, we could use these in the docs but I suspect that is overkill).